### PR TITLE
ignore patterns

### PR DESCRIPTION
**Pull request by LAShemilt**

### CODE REVIEW

File: rfi_file_monitor/applicationwindow.py [hunk 1]
```diff
@@ -256,6 +256,28 @@ def __init__(self, *args, **kwargs):
                 hexpand=True, vexpand=False,
         ), 'allowed_patterns')
         allowed_patterns_grid.attach(self._allowed_patterns_entry, 1, 0, 1, 1)
+        self._add_advanced_options_horizontal_separator()
```
Comment (tschoonj) [inline rfi_file_monitor/applicationwindow.py:259]:
```suggestion
        
        self._add_advanced_options_horizontal_separator()
        
```

Add newlines before and after to improve readability. I will be really happy when we can move the advanced options to its own class and file!

**Review by tschoonj — CHANGES_REQUESTED** [review]
Apart from the comment about additional newlines, looks good to me.

Thanks Laura!

**Comment by LAShemilt** [comment]
I did not! now I know why my code no longer works :)

File: setup.cfg [hunk 2]
```diff
@@ -41,7 +41,7 @@ rfi_file_monitor.operations =
     S3Uploader = rfi_file_monitor.operations.s3_uploader:S3UploaderOperation
     SftpUploader = rfi_file_monitor.operations.sftp_uploader:SftpUploaderOperation
     LocalCopier = rfi_file_monitor.operations.local_copier:LocalCopierOperation
-gui_scripts =
+console_scripts =
```
Comment (tschoonj) [inline setup.cfg:44]: This should also not have gone into the commit 😄

File: rfi_file_monitor/utils/__init__.py [hunk 3]
```diff
@@ -52,6 +52,12 @@ def get_patterns_from_string(input: str) -> List[str]:
     # multiple patterns are supported, provided they are separated with commas
     return list(map(lambda x: x.strip(), input.split(',')))
 
+def get_ignore_patterns_from_string(input: str, defaults=[]) -> List[str]:
```
Comment (tschoonj) [inline rfi_file_monitor/utils/__init__.py:55]: Is it possible to combine `get_patterns_from_string` and `get_ignore_patterns_from_string` into a single method?

**Pull request merged**